grails-app/views/order/_orderInvoices.gsp

```diff
@@ -26,7 +26,7 @@
                                 ${invoiceItem?.description}
                             </g:if>
                             <g:else>
-                                <format:displayName product="${invoiceItem?.product}" productSupplier="${invoiceItem?.orderItem?.productSupplier}" showTooltip="${true}" />
+                                <format:displayName product="${invoiceItem?.product}" productSupplier="${invoiceItem?.shipmentItem?.orderItem?.productSupplier}" showTooltip="${true}" />
```

Comment:
I remember you showing that the previous version (the one you are getting rid of) was displaying correctly for the prepayment items (there was `invoiceItem.orderItem` present).
Won't this change make the opposite situation, so that this will work for the rest now, but will not work for the prepayment items, as they do not have a `shipmentItem`? :thinking: